test/ec2-metadata-test-proxy/cmd/ec2-metadata-test-proxy.go

```diff
@@ -73,6 +90,18 @@ func getListenAddress() string {
 	return ":" + port
 }
 
+func initEnabledPaths() {
+	pathsEnabled = make(map[string]bool)
+	if strings.ToUpper(getEnv(spotInstanceActionFlag, "")) == "TRUE" {
```

Comment:
https://golang.org/pkg/strconv/#ParseBool :)

Reply:
function removed and used ParseBool instead of poor mans parsing in the new function :)